src/chainparams.h

```diff
@@ -14,6 +14,48 @@
 
 #include <vector>
 
+/** Default for -minrelaytxfee, minimum relay fee for transactions */
+static const unsigned int DEFAULT_MIN_RELAY_TX_FEE = 1000;
+//! -maxtxfee default
+static const CAmount DEFAULT_TRANSACTION_MAXFEE = 0.1 * COIN;
+//! Discourage users to set fees higher than this amount (in satoshis) per kB
+static const CAmount HIGH_TX_FEE_PER_KB = 0.01 * COIN;
+//! -maxtxfee will warn if called with a higher fee than this amount (in satoshis)
+static const CAmount HIGH_MAX_TX_FEE = 100 * HIGH_TX_FEE_PER_KB;
+/** Default for -maxorphantx, maximum number of orphan transactions kept in memory.
+ *  A high default is chosen which allows for about 1/10 of the default mempool to
+ *  be kept as orphans, assuming 250 byte transactions.  We are essentially disabling
+ *  the limiting or orphan transactions by number and using orphan pool bytes as
+ *  the limiting factor, while at the same time allowing node operators to
+ *  limit by number if transactions if they wish by modifying -maxorphantx=<n> if
+ *  the have a need to.
+ */
+static const unsigned int DEFAULT_MAX_ORPHAN_TRANSACTIONS = 1000000;
+/** BU's default for -limitancestorcount, max number of in-mempool ancestors */
+static const unsigned int BU_DEFAULT_ANCESTOR_LIMIT = 25;
+/** BU's default for -limitancestorsize, maximum kilobytes of tx + all in-mempool ancestors */
+static const unsigned int BU_DEFAULT_ANCESTOR_SIZE_LIMIT = 101;
+/** BU's default for -limitdescendantcount, max number of in-mempool descendants */
+static const unsigned int BU_DEFAULT_DESCENDANT_LIMIT = 25;
+/** BU's default for -limitdescendantsize, maximum kilobytes of in-mempool descendants */
+static const unsigned int BU_DEFAULT_DESCENDANT_SIZE_LIMIT = 101;
+
+/** Network default for the max number of in-mempool ancestors */
+static const unsigned int BCH_DEFAULT_ANCESTOR_LIMIT = 25;
```

Comment:
We currently know that `BCH_*_LIMIT` params are applied to chain of unconfirmed transactions in ABC client, what are we supposed to if those values are going to change? Adapt the code and recompile or just disable the feature va the tweak defined below? Wouldn't be better to have a way to expose those value as a configuration parameters.